### PR TITLE
Library association

### PR DESCRIPTION
**Pull request by ademibowale**
- I create a class Classroom
    - I use setter and getter for @label 
- I create a class Book
    - I use setters and getters for instance variable
- Create a class Rental  
    - Use setter and getter for @date
-  I implemented: 
    - I create the has-many side of Book and Rental (a book has many rentals).
    - I create the belongs-to side of Rental and Book (a rental belongs to a book).
    - I create the has-many side of Person and Rental (a person has many rentals).
    - I create the belongs-to side of Rental and Person (a rental belongs to a person).
    - I modify the constructor of Rental so Book and Person are set in it.

### CODE REVIEW

File: classroom.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label, :student
```
Comment (julie-ify) [inline classroom.rb]: - [ ] You defined `students`, instance not `student`. Kindly fix your typo error here.

File: README.md [hunk 2]
```diff
@@ -1 +1,67 @@
...
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+## Author
+
```
Comment (julie-ify) [inline README.md]: - [ ] You have used the right template and added related data for the author section. But you are still missing these other sections. Kindly add a proper title, description, prerequisites, and steps to run and run the project locally.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @ademibowale,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (ademibowale) [reply README.md]: Thank you so much i will do that now

File: classroom.rb (hunk 1 above)
Comment (ademibowale) [reply classroom.rb]: oh great thank for that its done

**Review by Wes-Isaac — APPROVED** [review]
Hi @ademibowale, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Professional README 👀.
- Great communication 👍.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**